Stratis.Bitcoin/Connection/DropNodesBehaviour.cs

```diff
@@ -23,18 +23,18 @@ public DropNodesBehaviour(ConcurrentChain chain, IConnectionManager connectionMa
             this.chain = chain;
             this.connection = connectionManager;
 
-            // 80% of current max connections, the last 20% will only 
-            // connect to nodes ahead of the current best chain
+            // 80 % of current max connections, the last 20 % will only 
```

Comment:
I think % attached to the number is fine.

Reply:
hmm, you are right https://english.stackexchange.com/a/3284 - english does not prescribe this
in my language there is a notation that "80%" means something else than "80 %"
i.e. it is correct to write "80% solution of water with whatever" but not "80 % solution of water with whatever" and vice versa, it is correct to write "80 % of all people do something" but not "80% of all people do something"

but english seems to be weird in that, so :/

Reply:
do you want me to revert that ?

Reply:
thanks for the explanation :-D
yes please :-)

Reply:
fixed back